examples/plot_clip.py

```diff
@@ -0,0 +1,102 @@
+"""
+Clip Vector Data with EarthPy
+==================================================================
+
+Learn how to clip point, line, or polygon geometries with a
+polygon geometry using EarthPy.
+
+"""
+
+###############################################################################
+# Clip Vector Data in Python Using EarthPy
+# ---------------------------------------------
+#
+# The example below walks you through a typical workflow for clipping one
+# vector data file to the shape of another. Both vector data files must be
+# opened with GeoPandas as GeoDataFrames and be in the same Coordinate
+# Reference System (CRS) for the ``clip_shp()`` function in EarthPy to work.
+# The object to be clipped will be clipped to the full extent of the clip
+# object.
+#
+# .. note::
+#    If there are multiple polygons in clip object, data from shp will be clipped
+#    to the total boundary of all polygons in clip object.
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages. You will primarily use EarthPy's clip
+# utility alongside GeoPandas.
+
+import os
+import matplotlib.pyplot as plt
+import geopandas as gpd
+import earthpy as et
+import earthpy.clip as ec
+
+###############################################################################
+# Import Example Data
+# -------------------
+#
+# Once the packages have been imported, download the data needed for this
+# example: one line shapefile containing all major roads in North America,
+# and one polygon shapefile containing the United States border.
+
+data = et.data.get_data("spatial-vector-lidar")
+
+###############################################################################
+# Open Files with GeoPandas and Reproject the Data
+# -------------------------------------------------
+#
+# The first step to clipping geometries with EarthPy is to open them with
```

Comment:
This paragraph feels a bit long. I suggest condensing the text a bit and making it more active, something like: 

"Start by setting your working directory. Then, import the data files to GeoDataFrames using GeoPandas. 

Recall that the data must be in the same CRS in order to use the ```clip_shp()``` function. If the data are not in the same CRS, be sure to use the ```to_crs()``` function from GeoPandas to match the projects between the two objects, as shown below."